db/migrate/20170913132317_create_default_topics.rb

```diff
@@ -0,0 +1,17 @@
+class CreateDefaultTopics < ActiveRecord::Migration[5.1]
+  def up
+    Topic.create(title: 'Football')
+    Topic.create(title: 'Travel')
+    Topic.create(title: 'Politics')
+    Topic.create(title: 'Art')
+    Topic.create(title: 'Dating')
+    Topic.create(title: 'Music')
+    Topic.create(title: 'Movies')
+    Topic.create(title: 'Series')
+    Topic.create(title: 'Food')
+  end
+
+  def down
+    Topic.delete_all
```

Comment:
Instead of deleting all, you should delete only the topics that the migration creates. In this case, the end result would be the same, but conceptually it's wrong.